src/createProject.js

```diff
@@ -0,0 +1,211 @@
+/* eslint-disable import/no-cycle */
```

Comment:
With the ESLint inline comment, we are indirectly changing the rules by disabling these rules, let's fix the codebase instead.

What could be the reason? If you are using VSCode, you can delete this line and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇